### ISSUE TITLE
File Structure of Prices_Train, Strategy_Train and Strikes_Train inquiry

### ISSUE THREAD

**Issue by lecolearning** [juliansester/Deep-Arbitrage#2]
Hi Julian, thanks for the open source contribution, have at question on the below code:

Prices_Train = pd.read_csv("csv/Prices_Train_real.csv", header=None)
Strategy_Train = pd.read_csv("csv/Strategy_Train_real.csv", header=None)
Strikes_Train = pd.read_csv("csv/Strikes_Train_real.csv", header=None)

do you have a example for the format? 

Also for Strategy_Train_real.csv (or Y input) : is this generated by another algorithm like LSIP algorithm from Neufeld et al. (2022a)? inferring that from the passage: "Relying on these samples, we compute, using the LSIP algorithm from Neufeld et al. (2022a), minimal super-replication strategies of the 0-payoff for each of the 50 000 samples. Of these 50 000 samples, we regard 5000 samples as a test set on which the neural network is not trained."

Also facing issue in loading the model_strat_real.h5 file due to tensorflow version difference, would be be able to share the package dependencies? 

Thanks once again for the contribution.
